HoloRepositoryUI/client/src/components/public/LandingPage.tsx

```diff
@@ -1,15 +1,114 @@
 import React, { Component } from "react";
 import { Link, RouteComponentProps } from "@reach/router";
+import {
+  Dropdown,
+  IDropdownOption,
+  TextField,
+  PrimaryButton,
+  IImageProps,
+  ImageFit,
+  Image
+} from "office-ui-fabric-react";
+import { Layout } from "antd";
+import appLogo from "../../assets/logo/2x/logo_and_font@2x.png";
+import MainFooter from "../shared/MainFooter";
+import { PropsWithContext, withAppContext } from "../shared/AppState";
+
+const options: IDropdownOption[] = [
```

Comment:
Could you add a comment that these are hard-coded because the authentication is just a proof-of-concept for now?